operations/mimir-mixin/docs/playbooks.md

```diff
@@ -446,6 +446,25 @@ How to **investigate**:
 
 - Look for any scan error in the store-gateway logs (ie. networking or rate limiting issues)
 
+### MimirStoreGatewayNoSyncedBlocks
+
+This alert fires a store-gateway is not syncing any blocks. Effectively it is sitting idle because no blocks are sharded to it.
+
+How it **works**:
+
+- Store-gateways join a hash ring to shard block IDs across all store-gateway replicas.
+- A tenant can be sharded across multiple store-gateways. How many exactly is determined by `-store-gateway.tenant-shard-size` or the `store_gateway_tenant_shard_size` limit.
+- When the tenant shard size is less than the replicas of store-gateways, some store-gateways may not get any tenants' blocks sharded to them.
```

Comment:
```suggestion
- When the tenant shard size is less than the replicas of store-gateways and a Mimir cluster has very few tenants, some store-gateways may not get any tenants' blocks sharded to them.
```

Reply:
I wanted to keep the sentence shorter, so I explained this in the next bullet point. Is that ok?

Reply:
Yes, sure.